python/paddle/v2/fluid/tests/book/test_recognize_digits_mlp.py

```diff
@@ -58,43 +55,35 @@
         y_data = np.array(map(lambda x: x[1], data)).astype("int64")
         y_data = np.expand_dims(y_data, axis=1)
 
-        tensor_x = core.LoDTensor()
+        tensor_x = fluid.LoDTensor()
         tensor_x.set(x_data, place)
 
-        tensor_y = core.LoDTensor()
+        tensor_y = fluid.LoDTensor()
         tensor_y.set(y_data, place)
 
-        outs = exe.run(framework.default_main_program(),
+        outs = exe.run(fluid.default_main_program(),
                        feed={'x': tensor_x,
                              'y': tensor_y},
                        fetch_list=[avg_cost] + accuracy.metrics)
         out = np.array(outs[0])
         acc = np.array(outs[1])
         pass_acc = accuracy.eval(exe)
 
-        test_accuracy = evaluator.Accuracy(input=predict, label=label)
+        test_accuracy = fluid.evaluator.Accuracy(input=predict, label=label)
 
         test_target = [avg_cost] + test_accuracy.metrics + test_accuracy.states
-        inference_program = get_inference_program(test_target)
+        inference_program = fluid.io.get_inference_program(test_target)
 
         test_accuracy.reset(exe)
         for data in test_reader():
             x_data = np.array(map(lambda x: x[0], data)).astype("float32")
             y_data = np.array(map(lambda x: x[1], data)).astype("int64")
             y_data = np.expand_dims(y_data, axis=1)
 
-            tensor_x = core.LoDTensor()
-            tensor_x.set(x_data, place)
-
-            tensor_y = core.LoDTensor()
-            tensor_y.set(y_data, place)
-
-            outs = exe.run(inference_program,
-                           feed={'x': tensor_x,
-                                 'y': tensor_y},
-                           fetch_list=[avg_cost] + test_accuracy.metrics)
-            out = np.array(outs[0])
-            acc = np.array(outs[1])
+            out, acc = exe.run(inference_program,
+                               feed={'x': x_data,
```

Comment:
So executor run interface will create LoDTensor and set numpy data?

Reply:
I see, we have a `aslodtensor` helper function now.

Reply:
Executor can only create Tensor now. LoDTensor needs to be set manually.